operations/host_spawn.go

```diff
@@ -76,6 +80,22 @@ func hostCreate() cli.Command {
 				Name:  noExpireFlagName,
 				Usage: "make host never expire",
 			},
+			cli.StringSliceFlag{
+				Name:  wholeWeekdaysOffFlagName,
+				Usage: "for an unexpirable host, the days when the host should be turned off for its sleep schedule (allowed values: Sunday, Monday, Tuesday, Wednesday, Thursday, Friday, Saturday)",
+			},
+			cli.StringFlag{
+				Name:  dailyStartTimeFlagName,
+				Usage: "for an unexpirable host, the time when the host should start each day for its sleep schedule",
```

Comment:
Do these only work with unexpirable hosts? Should we add a before function to make them mutually exclusive / inclusive like [here](https://github.com/ZackarySantana/evergreen/blob/385cf6859b8f87770bf99f2ed4dfa2316a6278cd/operations/patch.go#L98)?

Reply:
It should already be validated on the back end [here](https://github.com/evergreen-ci/evergreen/blob/4f83e140d27f8712189f937e2c4bd86959108097/cloud/spawn.go#L88-L90).